pkg/apis/servicecatalog/types.go

```diff
@@ -97,7 +94,7 @@ const (
 
 // ServiceClass represents an offering in the service catalog.
 type ServiceClass struct {
-	kunversioned.TypeMeta
```

Comment:
Interesting - I think I was wrong about this.  It has changed since I last looked closely at how most internal APIs were structured.